lib/rocket_job/job.rb

```diff
@@ -12,5 +12,6 @@ class Job
     include Plugins::Job::Worker
     include Plugins::Job::Throttle
     include Plugins::Job::ThrottleRunningJobs
+    include Plugins::Job::ThrottleDependantJobs
```

Comment:
Can we make this throttle optional? I.e. It can be included explicitly when a job wants this capability?
For example the `ThrottleWindows` throttle is explicitly included into jobs when needed.
Additionally, by including it here, `Batch` will not be loaded yet, so it will not load the batch throttle.